fabric-tool-attributes-v1/src/main/java/net/fabricmc/fabric/api/tool/attribute/v1/ToolActor.java

```diff
@@ -14,15 +14,25 @@
  * limitations under the License.
  */
 
-package net.fabricmc.fabric.mixin.mining.level;
+package net.fabricmc.fabric.api.tool.attribute.v1;
 
-import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.gen.Accessor;
+import edu.umd.cs.findbugs.annotations.Nullable;
```

Comment:
Do we have a consensus anywhere on which Nullable we are even going to use? I remember one discussion where people seemed to agree something like jetbrains was the preferred option but no firm decision from what I see anywhere else in the codebase.

Reply:
I think this might be the first Nullable in Fabric API, actually.

Reply:
This will be changed when #461 is merged